index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body class="container">
+    <header class="header">
+        <p class="name">Fernando Herrera</p>
```

Comment:
- [ ] Please ensure that this logo text uses an `a` tag.
Something like this would be really great. 
```suggestion
        <a href="/" class="name">Fernando Herrera</a>
```